hash.go

```diff
@@ -0,0 +1,46 @@
+package wolfSSL
+
+// #cgo CFLAGS: -g -Wall -I/usr/include -I/usr/include/wolfssl
+// #cgo LDFLAGS: -L/usr/local/lib -lwolfssl -lm
+// #include <wolfssl/options.h>
+// #include <wolfssl/wolfcrypt/hash.h>
+// #undef NO_MD5
```

Comment:
Instead of manually having this `#undef NO_MD5`, I think we should just let the value be picked up from the `<wolfssl/options.h>` include above.

Reply:
Nice catch! I was actually aware of the problem but it wasn't pulling the macro from options.h. Moved some things around and it works now (having a fresh set eyes helps).